index.html

```diff
@@ -5,130 +5,159 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js" defer></script>
+    <link rel="stylesheet" href="css/style.css">
+    <script src="javascript/script.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap" rel="stylesheet"> 
 </head>
 <body>
 
+    <section class="social-icons-section">
+        <ul class="social-icons">
+            <li class="network-icons-list"><a href="https://twitter.com/mkaya133" target="_blank"><img src="img/twitter.svg" alt=""></a></li>
+            <li class="network-icons-list"><a href="https://www.linkedin.com/in/mert-kaya-0732b717b/" target="_blank"><img src="img/linkedin.svg" alt=""></a></li>
+            <li class="network-icons-list"><a href="https://github.com/mkaya13" target="_blank"><img src="img/github.svg" alt=""></a></li>
+            <li>English</li>
+        </ul>
+    </section>
```

Comment:
- [ ] This isn't really a section I think that using the `section` tag is kind of wrong here, you can use either a `nav` tag or just a `div` tag, kindly adjust that.
- [ ] Please also follow the design provided, yours look like this now:
  
   ![image](https://user-images.githubusercontent.com/20567503/197232393-4034611b-e3fb-4c47-ac9f-469ac21971c6.png)

   The original design is like this:

   ![image](https://user-images.githubusercontent.com/20567503/197232810-e2c05c86-94a1-4593-8359-3274420e51dc.png)
   
   Kindly fix this.